docs/installation/quick-start.md

```diff
@@ -7,6 +7,11 @@ Bee is packaged for Ubuntu, Raspbian, Debian and CentOS based Linux distribution
 
 If your system is not supported, please see the [manual installation](/docs/installation/manual) section for information on how to install Bee.
 
+The overview of the installation process:
+ 1. set up an external signer for Bee (e.g. install *bee-clef*)
```

Comment:
```suggestion
 1. (Recommended) set up an external signer for Bee (i.e. install *bee-clef*)
```
Bee clef is optional, and at the moment the only option for an external signer, so have suggested a couple of changes 😉